cnf-certification-test/suite_test.go

```diff
@@ -168,7 +179,7 @@ func TestTest(t *testing.T) {
 
 	configurations, err := claimhelper.MarshalConfigurations()
 	if err != nil {
-		log.Errorf("Configuration node missing because of: %s", err)
+		log.Errorf("Configuration node missing because of: %v", err)
```

Comment:
Should the specifier be %w for error?

Reply:
You only need %w if the calling function needs to "unwrap" the underlying error.  As far as I know we don't do any of that.